lib/ember-build.js

```diff
@@ -29,6 +29,9 @@ function EmberBuild(options) {
 
   if (options && options.packages) {
     this._packages = options.packages;
+    this._name = options.name || 'ember';
```

Comment:
`this._name`, `this._namespace` and `this._skipRuntime` should be initialized as `null` in `init` method so object shape stays the same.